From e437da60918b1218489b98d8c9d15a5d8f186000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andrius=20=C5=A0tikonas?= <andrius@stikonas.eu>
Date: Sun, 12 Jun 2016 16:15:18 +0100
Subject: [PATCH 7/7] Also check whether HFS+ shrink support is available.

---
 src/fs/hfsplus.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/fs/hfsplus.cpp b/src/fs/hfsplus.cpp
index a37bc21..0ff6d85 100644
--- a/src/fs/hfsplus.cpp
+++ b/src/fs/hfsplus.cpp
@@ -53,13 +53,13 @@ bool hfsplus::supportToolFound() const
 {
     return
 //          m_GetUsed != cmdSupportNone &&
-         m_GetLabel != cmdSupportNone &&
+        m_GetLabel != cmdSupportNone &&
 //          m_SetLabel != cmdSupportNone &&
         m_Create != cmdSupportNone &&
         m_Check != cmdSupportNone &&
 //          m_UpdateUUID != cmdSupportNone &&
 //          m_Grow != cmdSupportNone &&
-//          m_Shrink != cmdSupportNone &&
+        m_Shrink != cmdSupportNone &&
         m_Copy != cmdSupportNone &&
         m_Move != cmdSupportNone &&
         m_Backup != cmdSupportNone;
-- 
2.8.3

