From 055b1fe3e50697b932a9729800a7e156ab619a17 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andrius=20=C5=A0tikonas?= <andrius@stikonas.eu>
Date: Sat, 12 Jan 2019 00:24:26 +0000
Subject: [PATCH] Make device model names prettier.

---
 src/plugins/sfdisk/sfdiskbackend.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/sfdisk/sfdiskbackend.cpp b/src/plugins/sfdisk/sfdiskbackend.cpp
index 66472b3..3daa188 100644
--- a/src/plugins/sfdisk/sfdiskbackend.cpp
+++ b/src/plugins/sfdisk/sfdiskbackend.cpp
@@ -170,7 +170,7 @@ Device* SfdiskBackend::scanDevice(const QString& deviceNode)
         if ( d == nullptr && modelCommand.run(-1) && modelCommand.exitCode() == 0 )
         {
             QString name = modelCommand.output();
-            name = name.left(name.length() - 1);
+            name = name.left(name.length() - 1).replace(QLatin1Char('_'), QLatin1Char(' '));
 
             if (name.trimmed().isEmpty()) {
                 // Get 'lsblk --output kname' in the cases where the model name is not available.
-- 
2.17.1

